tests/unit/test_main.py

```diff
@@ -384,6 +384,25 @@ def test_init_command(mock_init_command, args, expected_parameters):
     mock_init_command.assert_called_with(**expected_parameters)
 
 
+@pytest.mark.parametrize(
+    "args",
+    [
+        (
+            {
+                "addon_name": "foo/bar",
```

Comment:
That's an interesting test, should we allow `foo/bar` as an add-on name for the `ucc-gen init` command?

That's out of scope for this PR but worth checking our validations.

Reply:
Actually, we don't allow `foo/bar` as an addon-name, it's also mentioned in one of the [unit test case](https://github.com/splunk/addonfactory-ucc-generator/blob/develop/tests/unit/commands/test_init.py#L145)
In the recent push I had changed the name to `splunk_add_on_for_demo`.